### DISCUSSION TITLE
error when initiate with last step

### DISCUSSION THREAD

**Discussion by Armstock** [omkarcloud/google-maps-scraper#148] [General]
![screenshot-20240318-152631](https://github.com/omkarcloud/google-maps-scraper/assets/56221017/4dc762de-0f04-462e-a187-80f78c9c4789)
Can anyone help?

**Comment by marcuswoy** [comment]
At the moment I am not able to use the application on Windows or Linux. But with docker-compose.yml you'll be able to use it. In this case you need to install docker.

**Comment by Chetan11-dev** [comment]
Kindly Run Commands 
python -m pip install botasaurus --upgrade
python -m pip install botasaurus_server --upgrade

**Reply by Chetan11-dev** [reply]
Let me know if it resolves it, In case it does not kindly share screenshots.
